pypesto/startpoint/latin_hypercube.py

```diff
@@ -5,6 +5,23 @@
 def latin_hypercube(**kwargs) -> np.ndarray:
```

Comment:
```suggestion
def latin_hypercube(n_starts: int,
        lb: Union[np.ndarray, List[float]],
        ub: Union[np.ndarray, List[float]],
        smooth: bool) -> np.ndarray:
```
Would add the parameters directly. Seems more informative to me then kwargs. Or is there a special reason for `**kwargs`?

Reply:
The interface for the sampling routines are quite various, therefore all sampling routines have the `**kwargs` argument currently (since they are not completely standardized and may use different ones, depending on the methods...). Since this isn't the focus of this PR, I would leave this one to an other refactoring of the starting value sampling...

Reply:
perfectly fine with that :)